apps/minaintyg/src/pages/Certificate/CertificateListPage.tsx

```diff
@@ -31,11 +31,12 @@ export function CertificateListPage() {
       <CertificateListFilter listed={list?.content.length ?? 0} />
       <CertificateListOrder setOrder={setOrder} order={order} />
       {isLoading && <IDSSpinner data-testid="certificate-list-spinner" />}
-      {isError && (
-        <TechnicalIssueAlert>
+      {error && (
```

Comment:
Correct me if im wrong, but this is the result of a 500.

Since we dont wanna implement filters that generate id's from backend, the error.id needs to be generated in frontend?